packages/hmr-plugin/src/public_api.ts

```diff
@@ -1,4 +1,10 @@
-export { NgxsHmrLifeCycle, NgxsHmrOptions, NgxsHmrSnapshot } from './symbols';
 export { HmrInitAction } from './actions/hmr-init.action';
+export {
+  NgxsHmrLifeCycle,
+  NgxsHmrOptions,
+  NgxsHmrSnapshot,
+  WebpackModule,
+  BootstrapModuleFn
```

Comment:
Why have `WebpackModule` and `BootstrapModuleFn` been added to the exports?
Was something not working? It works fine in my app without this.

Reply:
no, it works without, I just wanted to use the types in the example, the user may also want this